### ISSUE TITLE
Fail to download. File not Found.

### ISSUE THREAD

**Issue by mmurooka** [wkentaro/gshell#9]
Failed to download even though the file can find with ls.
Maybe this problem began to occur after update within one month.

```
leus@cygnus:/media/leus/Murooka_1TB/compressed$ gshell ls
gazebo_object_motion_small_manip_10000.tgz
gazebo_object_motion_small_10000_split.tgz
gazebo_object_motion_small_10000.tgz
gazebo_object_motion_scalable_cube_2722.tgz
gazebo_object_motion_cf2_10000.tgz
gazebo_object_motion_10000_split.tgz
gazebo_object_motion_cf_10000.tgz
gazebo_object_motion_f_10000.tgz
gazebo_object_motion_c_10000.tgz
movie
gazebo_object_motion_v_10000.tgz
gazebo_object_motion_70000.tgz
gazebo_object_motion_10000.tgz
gazebo_object_motion_1
gazebo_object_motion_2000_20161025
gazebo_object_motion_2000
gazebo_object_motion_100
leus@cygnus:/media/leus/Murooka_1TB/compressed$ gshell download gazebo_object_motion_small_manip_10000.tgz
Failed to get file: googleapi: Error 404: File not found: None., notFound
```

**Comment by mmurooka** [comment]
find id of the file with web browser, and
```
gshell download --with-id ****
```
is working.

**Comment by wkentaro** [comment]
What version?

2016年12月27日(火) 11:37 Masaki Murooka <notifications@github.com>:

**Comment by mmurooka** [comment]
```
leus@murooka-dlbox:~$ gshell --version
gshell, version 5.0.1
```

**Comment by wkentaro** [comment]
I reproduced that symptom. Maybe caused by the long name. fixing...

**Comment by wkentaro** [comment]
Should be fixed now by #10.

**Comment by mmurooka** [comment]
Thank you!
